iBudget/spending/views.py

```diff
@@ -83,17 +84,18 @@ def group_limit(request):
     """
     if request.method == 'GET':
         user_id = request.user
```

Comment:
Are you sure that this is user_id in the request but not user object?

Reply:
Its not my task, this code was written by Ihor